### PR TITLE
Drop use of ctrlc-windows

### PR DESCRIPTION
**Pull request by mook-as**
The ctrlc-windows npm package contains a rust executable; rust by default depends on vcruntime140.dll.  This means that on machines without the correct MSVC redistributables installed, Rancher Desktop will fail to start due to failing to import the native module.

Luckily, what we need from it is simple enough that we can just re-implement the necessary functionality as part of wsl-helper.

### CODE REVIEW

File: src/go/wsl-helper/pkg/process/kill_windows.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+func Kill(pid int) error {
+	if pid < 1 {
+		return fmt.Errorf("cannot kill process: invalid pid: %d", pid)
+	}
+
+	// Try to re-attach to the default console
+	defer func() {
+		_, _, _ = freeConsole.Call()
+		_, _, _ = attachConsole.Call(0xFFFFFFFF)
```
Comment (adamkpickering) [inline src/go/wsl-helper/pkg/process/kill_windows.go:34]: It feels like the `0xFFFFFFFF` should be a constant in the `windows` package, but I can't find anything like this :cry:

**Review by adamkpickering — APPROVED** [review]
It's tempting to turn this into an open-source module!

I don't see any real problems, just a couple minor things. Feel free to merge without changing anything, but I'm happy to do another quick review if you want to make changes.

File: src/go/wsl-helper/pkg/process/imports_windows.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package process
+
+import "golang.org/x/sys/windows"
+
+var (
+	kernel32Dll           = windows.NewLazySystemDLL("kernel32.dll")
```
[the remark this answers is not in the document]
Comment (mook-as) [reply src/go/wsl-helper/pkg/process/imports_windows.go:22]: In effect it makes no difference: it just delays the error to when the function gets called, which means it returns when `FreeConsole` fails. But to the caller it still means the process exited abnormally (hopefully with a useful error message).

File: src/go/wsl-helper/pkg/process/kill_windows.go (hunk 1 above)
Comment (mook-as) [reply src/go/wsl-helper/pkg/process/kill_windows.go:34]: According to the [docs](https://learn.microsoft.com/en-us/windows/console/attachconsole) it should be `ATTACH_PARENT_PROCESS (DWORD)-1` but yeah.

File: src/go/wsl-helper/cmd/kill_process_windows.go [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+var killProcessViper = viper.New()
+
+// killProcessCmd is the `wsl-helper kill-process` command.
+var killProcessCmd = &cobra.Command{
+	Use:   "kill-process",
+	Short: "Kill a given process",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		err := process.Kill(killProcessViper.GetInt("pid"))
+		if err != nil {
```
[the remark this answers is not in the document]
Comment (adamkpickering) [reply src/go/wsl-helper/cmd/kill_process_windows.go]: I wasn't aware of that, good to know 👍

**Pull request merged**